common/minimal.py

```diff
@@ -42,22 +42,22 @@ def osname():
 def main():
     # Put variable declarations here
     variables = {
-        'RoomTemp': {
+        'Room Temp': {
```

Comment:
I see that it is a string. Why can't I use spaces in strings? What will break and in what cases? I don't think that saving one byte of data in each message worth worsening the readability and elegance of these messages.